src/components/itineraries/Generate.tsx

```diff
@@ -150,6 +157,14 @@ const Generate: FC = () => {
             <StyledInput type="number" value={ num } onChange={ onChangeNum } />
           </div>
         </Div>
+        <Div>
+            <Link to='?addAddress=1'>
```

Comment:
I've been using `src/config/page.ts` to method for this to property on Links. Maybe we could extend that function to support something like `<Link to={ to(undefined, { addAddress: 1 }) } />`

Reply:
Agreed to use hash.